pkg/codegen/nodejs/gen.go

```diff
@@ -45,11 +45,13 @@ import (
 	"github.com/pulumi/pulumi/sdk/v3/go/common/util/contract"
 )
 
-// The minimum version of @pulumi/pulumi compatible with the generated SDK.
 const (
-	MinimumValidSDKVersion   string = "^3.42.0"
-	MinimumTypescriptVersion string = "^4.3.5"
-	MinimumNodeTypesVersion  string = "^14"
+	// The minimum version of @pulumi/pulumi compatible with the generated SDK.
+	MinimumValidSDKVersion string = "^3.42.0"
+	// The minumum version of @pulumi/pulumi that supports parameterization.
+	MinimumValidParameterizationSDKVersion string = "^3.129.0"
```

Comment:
We have been reluctant to bump `MinimumValidSDKVersion` in the past. To sidestep that, I added a separate minimum version that's used for parameterized SDKs.